awscli/examples/configure/get/_examples.rst

```diff
@@ -24,12 +24,12 @@ The following commands would have the corresponding output::
     testing_access_key
 
     $ aws configure get profile.testing.aws_access_key_id
-    default_access_key
+    testing_access_key
 
     $ aws configure get preview.cloudsearch
     true
 
     $ aws configure get preview.does-not-exist
-    $
```

Comment:
This is showing that there is no output, resulting in the prompt immediately appearing. I would prefer to keep this.